README.md

```diff
@@ -175,8 +184,11 @@ Triage libFuzzer crashes with casr-libfuzzer:
 Triage Atheris crashes with casr-libfuzzer:
 
     $ unzip casr/tests/casr_tests/python/ruamel.zip
-    $ cp casr/tests/casr_tests/python/yaml_fuzzer.py .
-    $ casr-libfuzzer -i casr/tests/casr_tests/casrep/atheris_crashes_ruamel_yaml -o casr/tests/tmp_tests_casr/casr_libfuzzer_atheris_out -- ./yaml_fuzzer.py
+    $ casr-libfuzzer -i casr/tests/casr_tests/casrep/atheris_crashes_ruamel_yaml -o casr/tests/tmp_tests_casr/casr_libfuzzer_atheris_out -- casr/tests/casr_tests/python/yaml_fuzzer.py
```

Comment:
Have you tested this command? There were some import or paths problems as far as I remember.